docs/content/concepts/metadata-tags/op-job-metadata.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: "Op job metadata | Dagster Docs"
+description: Attach information to op jobs using metadata
+---
+
+# Op job metadata
+
+Metadata allows you to attach information to [op jobs](/concepts/ops-jobs-graphs/op-jobs) and the runs launched from those jobs. By the end of this guide, you'll understand what metadata is, how to attach it to an op job, and how to view the job metadata in the Dagster UI.
```

Comment:
@erinkcochran87 - one small thing I noticed about this page after the fact: says "in this guide", but it's a concept page.